docs/upgrade.md

```diff
@@ -267,3 +295,57 @@ toc:
   - file: myfolder/page1
   - file: myfolder/page2
 ```
+
+## FAQ: Migration and Troubleshooting
+
+(known-limitations)=
+### What are the known limitations in Jupyter Book 2?
+
+Some features from Jupyter Book 1 are still in development for version 2:
+
+**Not yet available:**
+- **Editable Thebe cells** - Interactive code cells that users can edit and execute ([#443](https://github.com/jupyter-book/mystmd/issues/443))
+- **API documentation** - Sphinx autodoc-style API documentation generation ([#1259](https://github.com/jupyter-book/mystmd/issues/1259))
+- **Embed cell outputs as MyST** - Generate MyST from notebook outputs ([#2114](https://github.com/jupyter-book/mystmd/issues/2114))
+
+**Available with different syntax:**
+- Most MyST directives (syntax mostly compatible)
+- Cross-references (new MyST syntax)
+- Themes and styling (new theming system)
+
+See [upgrade discussions](https://github.com/orgs/jupyter-book/discussions/categories/upgrading-jupyterbook) for specific features and workarounds.
+
+### Where did [feature X] from Jupyter Book 1 go?
```

Comment:
Did you say anywhere that it's not really about feature removals but about features not yet reimplemented? I think that is really important to emphasise over and over.